src/common/DataObjectsTable/DataObjectsTableCell.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import _ from 'lodash';
+import Moment from 'moment';
+
+import { FontIcon } from 'material-ui';
+
+const DataObjectsTableCell = ({ item, columnId, users }) => {
+  const renderDate = (value) => {
+    const date = Moment(value).format('DD/MM/YYYY');
+    const time = Moment(value).format('LTS');
+    const title = `${date} ${time}`;
+
+    return (
+      <div title={title}>
+        {date}
+      </div>
+    );
+  };
+
+  const renderFile = (value) => {
+    const handleFileClick = (event) => {
+      event.preventDefault();
+      event.stopPropagation();
+      window.open(value.value, '_blank');
+    };
+
+    return (
+      <div onClick={handleFileClick}>
+        <FontIcon className="synicon-download" />
+      </div>
+    );
+  };
+
+  const renderReference = (value) => (
+    <div>{`${value.target}: ${value.value}`}</div>
+  );
+
+  const objectTypeRenderMethods = {
+    datetime: (value) => renderDate(value.value),
+    file: (value) => renderFile(value),
+    reference: (value) => renderReference(value),
+    relation: (value) => renderReference(value)
+  };
+
+  const renderContent = (content) => {
+    if (!content) {
+      return null;
+    }
+
+    let renderedContent = content;
```

Comment:
instead you should just `return` in proper place instead assigning it to temporary variable